L.Map.Sync.js

```diff
@@ -41,6 +48,27 @@
             return this;
         },
 
+        // Helper function to compute the offset easily
+        // The arguments are relative positions respect refence and target maps
+        // of the point to sync.
+        // If you provide ratioRef=[0,1], ratioTgt=[1,0] will sync
+        // the bottom left corner of the reference map
+        // with the top right corner of the target map.
+        // The values can be less than 0 or greater than 1. It will sync
+        // points out of the map.
+        offsetHelper: function (ratioRef, ratioTgt) {
```

Comment:
Not sure if this method should be at home in this location, I mean, it doesn't interact with the instance at all. Is `L.Util` a better location?